src/databricks/labs/remorph/reconcile/query_builder/base.py

```diff
@@ -162,3 +178,48 @@ def _get_custom_transformation(
             transform_rule_mapping.append(TransformRuleMapping(col_origin, transform, col_alias))
 
         return transform_rule_mapping
+
+    @staticmethod
+    def _get_threshold_select_function(match_type, mode):
+        try:
```

Comment:
Wouldn't this be more readable as:
```
if match_type in Constants.NUMBER_TYPES:
  if mode == ThresholdMode.ABSOLUTE.value:
    return ThresholdSQLTemplate.SELECT_NUMBER_ABSOLUTE.value
  elif mode == ThresholdMode.PERCENTILE.value
    return ThresholdSQLTemplate.SELECT_NUMBER_PERCENTILE.value
elif match_type in Constants.DATETIME_TYPES:
  if mode == ThresholdMode.ABSOLUTE.value:
    return ThresholdSQLTemplate.SELECT_DATETIME.value
else:
  error_message = f"Invalid threshold match type or mode in get_threshold_select_function: {match_type} or mode: {mode}"
  logger.error(error_message)
  raise ValueError(error_message)
```
?

Reply:
@ganeshdogiparthi-db: This should be replaced by this ideally, https://sqlglot.com/sqlglot/expressions.html#DataType

@vil1: Welcome to the team 👍🏼

Reply:
Done. changed it to If else. Waiting on inputs from Sundar on how to setup expressions. Because pure sqlglot expressions are soo huge and they are difficult to maintain.